models/tracker/src/migration.ts

```diff
@@ -178,6 +203,23 @@ async function upgradeIssueStatuses (tx: TxOperations): Promise<void> {
   }
 }
 
+async function upgradeIssueTimeReportSettings (tx: TxOperations): Promise<void> {
+  const issues = await tx.findAll(tracker.class.Issue, {
+    defaultTimeReportDay: { $exists: false },
+    workDayLength: { $exists: false }
+  })
+
+  const teams = await tx.findAll(tracker.class.Team, { $in: Array.from(new Set(issues.map((issue) => issue.space))) })
```

Comment:
Top level $in is not supported, please use 
```suggestion
  const teams = await tx.findAll(tracker.class.Team, { _id: { $in: Array.from(new Set(issues.map((issue) => issue.space))) }})
```

Reply:
Done.